docker/docker-compose.dev.yml

```diff
@@ -28,7 +28,7 @@ x-server-build: &server-common
 services:
   immich-server:
     container_name: immich_server
-    command: ['/usr/src/app/bin/immich-dev', 'immich']
+    command: ['/usr/src/app/bin/immich-dev']
     <<: *server-common
```

Comment:
We can now clean up this server-common block as well

Reply:
Do you mean just inline it here now?

Reply:
Yea good spot